keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlVerificationRepository.kt

```diff
@@ -170,13 +170,27 @@ class SqlVerificationRepository(
         .insertInto(VERIFICATION_STATE)
         .set(VERIFICATION_STATE.STATUS, status)
         .set(VERIFICATION_STATE.METADATA, metadata)
-        .set(status.timestampColumn, currentTimestamp())
```

Comment:
On second thought, shouldn't this be set only in the insert (never in an update)? BTW, I didn't quite follow the comment on the duplicate key, since the insert would never even run without a value for the `started_at` column. My guess is that it passed in the tests because the way you set them up you'd never do an insert where that column would not be set (the first call to `updateState` always has a non-complete status, so the value was set).

Reply:
Wrong line, should be on line below.